pyanaconda/payload/dnfpayload.py

```diff
@@ -380,35 +382,39 @@ def _add_repo(self, ksrepo):
         #     kickstarts.
         if repo.id in self._base.repos:
             if not url and not mirrorlist and not metalink:
-                self._base.repos[repo.id].enable()
+                if enabled:
+                    self._base.repos[repo.id].enable()
```

Comment:
Isn't this repo enable useless when you will call it anyway in the 
```
if enabled:
    self._base.repos[repo.id].enable()
```
a few lines below?

Reply:
I believe it probably is, but I was not sure how best to document the behaviour being captured by the if statements above.  I'm open to ideas there.